R/coerce_to_messydate.R

```diff
@@ -11,90 +20,100 @@
 #' as_messydate("2021-02-01")
 #' as_messydate("20-01-2021")
 #' as_messydate("01-20-2021")
+#' as_messydate("20 September 2021")
 #' as_messydate("2021-02-01?")
 #' as_messydate("2021-02-01~")
 #' as_messydate("2021-02-01%")
 #' as_messydate("2021-02-01..2021-02-28")
 #' as_messydate("{2021-02-01,2021-02-28}")
+#' as_messydate("Second of February, two thousand and twenty-two")
+#' #as_messydate("01-02-21", resequence = TRUE)
 #' @export
-as_messydate <- function(x) UseMethod("as_messydate")
+as_messydate <- function(x, resequence) UseMethod("as_messydate")
```

Comment:
Best give the resequence argument a default parameter, even if it is just `NULL`

Reply:
I have added the parameter FALSE across for consistency, thank you